app/helpers/reference_helper.rb

```diff
@@ -0,0 +1,38 @@
+module ReferenceHelper
+
+  def after_desired_date?
+    Time.zone.today > Date.parse('2015-12-31')
+  end
+
+  def table_header
+    applicant = t('processed_applications.table_header.applicant')
+
+    if after_desired_date?
+      reference = t('processed_applications.table_header.reference')
+      "<th> #{reference} </th>\n<th> #{applicant} </th>".html_safe
+    else
+      "<th> #{applicant} </th>".html_safe
+    end
+  end
+
+  def table_column(application)
+    prefix = "<td> <a href=\"/#{controller.controller_name}/#{application.id}\">"
```

Comment:
Not insecure, but not safe in terms of result, because routes != controller names. But yeah, if we make sure it's gone in 3 weeks, I'm not gonna argue.